docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -181,7 +181,7 @@ The following sidebar will be generated:
 				{ label: 'Components', link: '/guides/components/' },
 				{ label: 'Internationalization (i18n)', link: '/guides/i18n/' },
 				{
-					label: 'advanced',
+					label: 'Advanced',
```

Comment:
This is a 1:1 match of the `advanced/` directory from line 169 as the entire "Guides" group from this section is autogenerated.